### DISCUSSION TITLE
Trying to change Icon for state of elk alarm sensor for garage doors.

### DISCUSSION THREAD

**Discussion by chrkov** [marcokreeft87/room-card#191] [Q&A] [answered]
Trying to get my icons to change for my garage door based on current state.   My garage doors are connected to an elk panel and report as expected.   I have this working with button cards, but trying to get it to work with room cards also.

Here is my code:

`type: custom:room-card
title: Garage Doors
show_icon: false
entities:
  - entity: sensor.elkm1_main_garage_door
    name: Main
    show_icon: true
    icon:
      conditions:
        - icon: mdi:garage
          condition: equals
          attribute: state
          value: Normal
        - icon: mdi:garage-open
          condition: equals
          attribute: state
          value: Violated
    tap_action:
      action: call-service
      service: scene.turn_on
      service_data:
        entity_id: scene.elkm1_main_garage_door`

Clicking on the icon works for opening and closing it.  But the icon itself never changes from the default.   I am sure I am doing something dumb..

Thanks.

**Comment by marcokreeft87** [comment, marked as answer]
State isn't an attribute. Try it without the attribute line

**Reply by chrkov** [reply]
That was it.   Thanks!
